### DISCUSSION TITLE
Sega System 1 compactibility

### DISCUSSION THREAD

**Discussion by Hikmori7** [djyt/system16_sprite_viewer#4] [Q&A]
Hi.
I'd plan to rip a Sega System Roms at the time being, but I had a question after read the Readme file.
Is your Sega System 16 sprite viewer program can able to rip Sega System 1 Rom main sprites?

**Comment by djyt** [comment]
Unfortunately not - it's a different format. Maybe in the future I'll add more hardware to the tool :)
